lib/xcpretty/formatters/rspec.rb

```diff
@@ -5,6 +5,7 @@ class RSpec < Formatter
     FAIL = "F"
     PASS = "."
     PENDING = "P"
+    MEASURING = "T"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.